action/awsconfig.go

```diff
@@ -87,9 +89,17 @@ func (c *ClusterConfig) Write(filePath string) (State, error) {
 	if err != nil {
 		return Error, err
 	}
-	_, err = section.NewKey("source_profile", viper.GetString("destination"))
-	if err != nil {
-		return Error, err
+
+	if c.Destination != "" {
+		_, err = section.NewKey("source_profile", c.Destination)
+		if err != nil {
+			return Error, err
+		}
+	} else {
+		_, err = section.NewKey("source_profile", viper.GetString("destination"))
+		if err != nil {
+			return Error, err
+		}
```

Comment:
you can use a local variable `destinationProfile` to get rid of some code duplication. something like 
```golang
destinationProfile := viper.GetString("destination")
if c.Destination != "" {
  destinationProfile := c.Destination
}
_, err = section.NewKey("source_profile", destinationProfile)
if err != nil {
	return Error, err
}
```

Reply:
used your code as is..